doc/doap.xml

```diff
@@ -0,0 +1,745 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<?xml-stylesheet href="doap.xsl" type="text/xsl"?>
```

Comment:
@lnj `href` must be adjusted too if you change the file's path.

Reply:
isn't `doap.xsl` also installed into the same location as `doap.xml`?

Reply:
Ah, it is OK then.